packages/wallets/src/features/cashier/modules/Transfer/hooks/useTransferMessages/utils/tradingPlatformStatusMessageFn.ts

```diff
@@ -0,0 +1,17 @@
+import { TRADING_PLATFORM_STATUS } from '../../../../../../cfd/constants';
+import { TMessageFnProps } from '../../../types';
+
+const tradingPlatformStatusMessageFn = ({ platformStatus }: TMessageFnProps) => {
+    const getStatusBadgeText = () => {
+        if (platformStatus === TRADING_PLATFORM_STATUS.MAINTENANCE) {
+            return 'We’re currently performing server maintenance, which may continue until 03:00 GMT. Please expect some disruptions during this time.';
```

Comment:
The hard coded time should be removed. This can be changed to temporarily just like server unavailable message.